crates/sui-types/src/sui_system_state.rs

```diff
@@ -70,6 +72,20 @@ pub struct Validator {
     pub delegation_staking_pool: StakingPool,
 }
 
+impl Validator {
+    pub fn to_current_epoch_committee_with_net_addresses(
+        &self,
+    ) -> (AuthorityName, StakeUnit, Vec<u8>) {
+        (
+            // TODO: Make sure we are actually verifying this on-chain.
+            AuthorityPublicKeyBytes::from_bytes(self.metadata.pubkey_bytes.as_ref())
+                .expect("Validity of public key bytes should be verified on-chain"),
+            self.stake_amount + self.delegation_staking_pool.epoch_starting_sui_balance,
```

Comment:
@emmazzz  / @lxfind  can you help me check the sum of these two things correctly represent the voting powers of a validator?

Reply:
That's correct.